packages/keep_sorted/project.bri

```diff
@@ -0,0 +1,62 @@
+import * as std from "std";
+import git, { gitCheckout } from "git";
+import { goBuild } from "go";
+
+export const project = {
+  name: "keep-sorted",
```

Comment:
The package name should match the directory name (see: https://github.com/brioche-dev/brioche/issues/67)

```suggestion
  name: "keep_sorted",
```